src/Microsoft.AspNet.Mvc.TagHelpers/js/LinkTagHelper_FallbackJavaScript.js

```diff
@@ -16,9 +16,12 @@
         scriptElements = doc.getElementsByTagName("SCRIPT"),
         // Find the meta tag before this script tag, that's the element we're going to test the CSS property on
         meta = scriptElements[scriptElements.length - 1].previousElementSibling,
+        // Get the current style of the meta tag starting with standards-based API and falling back to <=IE8 API
+        metaStyle = (doc.defaultView && doc.defaultView.getComputedStyle) ? doc.defaultView.getComputedStyle(meta)
+            : meta.currentStyle,
```

Comment:
nits:
- comment is a bit odd since the code contains a condition, not a start.  suggest "preferring the standards-based API (if available) and ..."
- will be slightly more readable if you spread the ternary onto three lines or write out the equivalent `if`
- most of the code leaves operators at the end of lines when wrapping